shoes-core/lib/shoes/color.rb

```diff
@@ -61,11 +61,7 @@ def black?
 
     def <=>(other)
       raise_class_mismatch_error(other) unless other.is_a?(self.class)
-      if same_base_color?(other)
-        @alpha <=> other.alpha
-      else
-        less_or_greater_than other
-      end
+      [red, green, blue, alpha] <=> [other.red, other.green, other.blue, other.alpha]
```

Comment:
This seems rather neat, it breaks behavior (that isn't fully caught in specs) though, or at least I believe so. 

With the current code, comparing [255, 0, 0] and [110, 110, 110] would result in the latter being better, with this the first one is bigger. 

Saying when a color is bigger than the other is hard business, the way we implement it right now is hoe shoes3 does it.

Your solution certainly is more elegant. 

I'm unsure here.